algorithmic_efficiency/workloads/fastmri/fastmri_jax/models.py

```diff
@@ -122,8 +125,8 @@ def __call__(self, x, train=True):
       output = jnp.concatenate((output, downsample_layer), axis=-1)
       output = conv(output, train)
 
-    output = final_conv(output)
-
+    out_channels = 1
```

Comment:
Why is this not a class attribute anymore?

Reply:
I forked this part from the old variants branch. I'm honestly not sure. Thanks for catching this, fixing.